ddtrace/tracing/trace_handlers.py

```diff
@@ -80,18 +80,25 @@ def _on_context_started(ctx):
 
 
 def _make_block_content(ctx, construct_url):
+    from ddtrace.appsec import _constants as _asm_constants
```

Comment:
Including an `appsec` import in the `tracing` directory confuses the separation of concerns between the Appsec and Tracing products. I recommend refactoring such that no appsec imports are necessary in this file.

Reply:
But this function is only called and used for ASM and defining ASM constants outside of ASM is also confusing for the separation of products.

Reply:
Same problem here, `_make_block_content` is an ASM only function that is used in the function `listen`.